packages/docs-website/docs/protocol-docs/client-specification/json-rpc-api.md

```diff
@@ -0,0 +1,420 @@
+---
+id: json-rpc-api
+title: JSON RPC Specification
+sidebar_label: JSON RPC Specification
+---
+
+In this section we provide an overview of the JSON RPC schema for the Nitro protocol. All methods accept requests and return responses following the [JSON RPC 2.0 specification](https://www.jsonrpc.org/specification).
+
+## CreateChannel
+
+This method is used to create a channel between a set of participants with given budgets. It contains information regarding type of channel and initial state of that channel.
+
+There are three different types of channels:
+
+- _Direct Channel_: channels that support applications between ledger channel counterparties
+- _Ledger Channel_: channels that manage the budgets and allocations between direct and virtual channels
+- _Virtual Channel_: channels that support applications routed over ledger channels
```

Comment:
This is pretty close, but I think it is better to think of channels as either being __application__ channels, or __ledger__ channels. Namely, they either run an application, or they [don't](https://docs.statechannels.org/docs/implementation-notes/consensus-app#implicit-or-null-app).

Separately, there are three main ways of funding a channel. 
1. Directly, with an on-chain deposit
2. Indirectly (sometimes called 'ledger funding'), via a ledger channel allocating funds to this one. That ledger channel might well be directly funded. 
3. Virtually, also via a ledger channel allocating funds to this one, but in a more complicated way*

* Namely, by a certain construction of several channels that fund each other and are participated in by a hub (in addition to one or the other or all of the participants of this channel). This is not yet documented nicely, and would be a nice thing to add (following the nitro paper but looking something like [this](https://docs.statechannels.org/docs/protocol-tutorial/off-chain-funding#indirect-funding)). 

I would also probably keep budgets out of this section, for clarity.